SampleViewer/mainSample.cpp

```diff
@@ -302,10 +302,17 @@ int main(int argc, char *argv[])
 #endif // QT_WEBVIEW_WEBENGINE_BACKEND
 
   QGuiApplication::setApplicationName("ArcGIS Maps Qt Samples");
-
   QGuiApplication::setOrganizationName("Esri");
   QApplication app(argc, argv);
-
+  
+  const auto fontPath = qEnvironmentVariable("ESRI_AUTOMATION_TEST_SUSE_FONT_PATH");
```

Comment:
`QtGlobals` is already included.

Reply:
I think we should check https://doc.qt.io/qt-6/qtenvironmentvariables-qtcore-proxy.html#qEnvironmentVariableIsSet first to see if it's set. Should we also put this check behind an ifdef for Linux only?